css/style.css

```
@@ -0,0 +1,112 @@
+
+body{
```

```suggestion
body {
```
Remember about space after the selector.